cookbooks/bcpc-hadoop/templates/default/hdp_hbase-regionserver-initd.erb

```diff
@@ -194,18 +194,26 @@ multi_hbase_daemon() {
         exit 1
     fi
 
-    export HBASE_${UPPERCASE_HBASE_DAEMON}_OPTS=" "
-
     for OFFSET in ${OFFSETS} ; do
+        export HBASE_${UPPERCASE_HBASE_DAEMON}_OPTS=" "
```

Comment:
This is from the original BIGTOP script. Wanted to make minimal change not knowing the history and also knowing that we will be using HDP provided scripts.